moredhel-keyvalue-resource.yml

```diff
@@ -0,0 +1,6 @@
+name: keyvalue
+repo: https://github.com/moredhel/keyval-resource
+container_image: moredhel/keyval-resource
+description: |
+  An alternate implementation the Key Value resource, but conforms to the Kubernetes configmap pattern
+  [Link to inspiration](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#add-configmap-data-to-a-specific-path-in-the-volume)
```

Comment:
hey @moredhel,
thanks for moving the PR to here :) 
We don't currently render the markdown links on the website. However, I think the description shouldn't need that since we're intending to keep the description as a short piece of info just describing what the resource does and keeping the extra info inside the resources readme (that at some point would be rendered on the website).

How do you feel about removing this and keeping it in the repo's readme?

Reply:
That's fine by me, I'll strip it now

Reply:
thanks